app/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "cast",
+  "version": "0.1.0",
+  "description": "A podcast listening PWA app",
```

Comment:
[nitpick (non-blocking)](https://conventionalcomments.org/#labels): "PWA app" repeats the word "app" twice

Reply:
fixed